src/Stripe.net/Services/OAuth/OAuthTokenService.cs

```diff
@@ -19,7 +19,7 @@ internal OAuthTokenService(ApiRequestor requestor)
         {
         }
 
-        [Obsolete("Please use the constructor that accepts an `ApiRequestor` instead.")]
+        [Obsolete("This member is deprecated and will be removed in a future release")]
```

Comment:
Why is this comment different from the others where we are marking the constructor that takes IStripeClient as obsolete?